pipeline/enigma/enigma-processing.py

```diff
@@ -133,9 +130,27 @@ def HGVS_cDNA_to_protein(hgvs_c):
     return hgvs_p
 
 
+def HGVS_p_to_AA_abrev(HGVS_p):
```

Comment:
Great!  This addresses issue #184 !  In enigma_postprocess.py, we can remove the function fixBrokenAminoAcidChange(), and simplify the code base that little bit more.